org.eclipse.ice.modeling/src/main/java/org/eclipse/ice/modeling/TestDriver_WF.java

```diff
@@ -0,0 +1,26 @@
+package org.eclipse.ice.modeling;
+
+import org.eclipse.ice.modeling.workflowEngine.*;
+import org.eclipse.ice.modeling.actors.*;
+
+public class TestDriver_WF {
+	/**
+	 * @param args
+	 */
+	public static void main(String[] args) {
```

Comment:
Why is this public static void main()? Why will this need to be run on its own?

Reply:
Just for my testing.  This should disappear once I learn more about how this is going to run and how to make that happen.

Reply:
Can you make this an actual junit test?